### DISCUSSION TITLE
VR/AR app integration

### DISCUSSION THREAD

**Discussion by violetazv** [EloiStree/EloiStree#2] [General]
Hi Éloi, I got recommended to contact you, as I am looking for a unity developer to integrate built scenes/components into a VR/AR app. Happy to connect and discuss further.

**Comment by EloiStree** [comment]
@violetazv  Hello, unfortunately,  I accepted a job and I am working on my current big program OMI.
Meaning that I am totally not available for the next two years.
Do you need just some quick help on a subject ? 
(If yes: what ? We never know that I can help you or redirect you)

Kind regards,
Eloi


( Note is will be more easy to communicate by Discord if you need to continuation this discussion. Feel free to ping me there if you need:
https://eloistree.page.link/discord )
